packages/block-library/src/file/deprecated.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+// Version of the file block without PR#28062 accessibility fix.
+const deprecated = [
```

Comment:
We're making changes to how the block is edited and saved, so we need to deprecate the previous version of the block. That way, it will still be supported in older posts.